src/components/Select/components/SelectMenu/SelectMenu.tsx

```diff
@@ -40,15 +44,54 @@ const SelectMenu = forwardRef<HTMLUListElement, SelectMenuProps>((props, ref) =>
   const renderOptions = () =>
     filteredOptions.length > 0 ? (
       <List
+        label={uniqueId('menu_list')}
         ref={combinedRefs}
-        data={filteredOptions}
-        rowSize={'small'}
+        height={5 * 40}
```

Comment:
make this constant

Reply:
the outcome is always the same so I don't see the performance implication. I will only change to give some context by naming

Reply:
https://github.com/Orfium/orfium-ictinus/pull/655#discussion_r1279165881

Reply:
fixed [`0363daa` (#655)](https://github.com/Orfium/orfium-ictinus/pull/655/commits/0363daa1f1c4208af7d7f8f3cbb6759ff638aaeb)